Kernel/Syscalls/execve.cpp

```diff
@@ -727,9 +736,14 @@ static Array<ELF::AuxiliaryValue, auxiliary_vector_size> generate_auxiliary_vect
         { ELF::AuxiliaryValue::Gid, (long)gid.value() },
         { ELF::AuxiliaryValue::EGid, (long)egid.value() },
 
+#if ARCH(I386) || ARCH(X86_64)
         { ELF::AuxiliaryValue::Platform, Processor::platform_string() },
         // FIXME: This is platform specific
         { ELF::AuxiliaryValue::HwCap, (long)CPUID(1).edx() },
+#elif ARCH(AARCH64)
+        { ELF::AuxiliaryValue::Platform, Processor::platform_string() },
+        { ELF::AuxiliaryValue::HwCap, 0L }, // TODO
```

Comment:
TODO: get HwCap from (???)

Reply:
processor? thereʼre cpuid counterparts… but that might be to specific (i still planned to use that information). got some ideas on what should i do here exactly?